docs/guides/libraries.rst

```diff
@@ -82,25 +82,25 @@ A typical directory structure would look like:
 
 .. code-block:: text
 
-   setup.py
+   pyproject.toml
    my_great_package/
       __init__.py
       stuff.py
       py.typed
 
 It's important to ensure that the ``py.typed`` marker file is included in the
-distributed package. If using ``setuptools``, this can be achieved like so:
```

Comment:
This note is afaik only relevant for setuptools anymore. Since the other build backends I'm aware support this in the default configuration, I've shortened it to this hint, the specifics fit better in the setuptools documentation.

Reply:
I shrunk the note to mentioning it needs to be included and all modern package managers do it by default, i.e. don't put it in your gitignore where hatchling skips it.